src/sagemaker/jumpstart/estimator.py

```diff
@@ -112,6 +112,7 @@ def __init__(
         disable_output_compression: Optional[bool] = None,
         enable_remote_debug: Optional[Union[bool, PipelineVariable]] = None,
         config_name: Optional[str] = None,
+        inference_config_name: Optional[str] = None,
```

Comment:
this should not be exposed as a constructor argument imo

Reply:
Agreed, there is better way to do this. I moved to only expose in deploy method